bootstrap-validator/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for bootstrap-validator 0.11.8
+// Project: https://github.com/1000hz/bootstrap-validator
+// Definitions by: Brady Liles <https://github.com/BradyLiles/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="jquery"/>
+
+interface ValidatorOptions {
+    delay?: number;
+    html?: boolean;
+    disable?: boolean;
+    focus?: boolean;
+    feedback?: any;
+    custom?: any;
+}
+
+interface JQuery {
+    validator(options?: ValidatorOptions): JQuery;
+    validator(command: string): JQuery;
+}
+
+declare module "bootstrap-validator" {
```

Comment:
If `bootstrap-validator` is just a jquery plugin that has no exports of its own, remove this.
If it has exports, describe them.